docs/development_suite/api-console/advanced-section/dev-console-config.md

```diff
@@ -1,9 +1,44 @@
 ---
-id: custom-ca-certs
-title:  How to add custom CA certs to core services
-sidebar_label: CA certs to core services
+id: dev-console-config
+title:  Console Configuration
+sidebar_label: Console Configuration
 ---
 
+In the sub-section `api-console-config` there's an empty `core-service.json` that can override the information of your micro-service.
+
+## Change the name of core services
+
+If you want to change the name of a service you will need to compile a json file with the structure similar to the one shown below:
+
+```
+{
+    "ag-1": {
```

Comment:
let's remove this service example since is not a core service from years, anymore